### PR TITLE
add Fonts.addIcon, Fonts.setIcon

### PR DESCRIPTION
**Pull request by DeltaNedas**
example usage:
```
Fonts.setIcon("egg", <some unicode char>, Core.atlas.find("egg"));
// Usable by the sector icon picker
Fonts.addIcon("gnu", "not-unix");
```

### CODE REVIEW

**Comment by DeltaNedas** [comment]
@Anuken travis

File: core/src/mindustry/ui/Fonts.java [hunk 1]
```diff
@@ -102,10 +103,51 @@ public static TextureRegion getLargeIcon(String name){
         });
     }
 
+    public static void setIcon(String name, int ch, TextureRegion region){
+        //ensure that modded icons don't interfere with existing ones; put them arter vanilla
```
Comment (Gdeft) [inline core/src/mindustry/ui/Fonts.java:107]: *Did you mean **after**?*

Comment (DeltaNedas) [reply core/src/mindustry/ui/Fonts.java:107]: maybe

**Comment by Anuken** [comment]
This doesn't work.

**Comment by DeltaNedas** [comment]
ok will probably try again in a few months/years

**Pull request closed without merging**